docker-app/qfieldcloud/core/admin.py

```diff
@@ -633,12 +633,21 @@ def lookups(self, request, model_admin):
             ("not finalized", _("not finalized")),
         )
 
-    def queryset(self, request, queryset):
-        q = Q(status="pending") | Q(status="started") | Q(status="queued")
-        if self.value() == "not finalized":
-            return queryset.filter(q)
+    def queryset(self, request, queryset) -> QuerySet:
+        value = self.value()
+
+        if value is None:
+            return queryset
+
+        not_finalized = Q(status="pending") | Q(status="started") | Q(status="queued")
```

Comment:
Last thing, which I spotted only now. Is there a reason to prefer to string representation that using `Job.Status` enum?

You can either keep it with multiple `Q(status=Job.Status.PENDING) | ...` or write it in a single line with `Q(status__in=(Job.Status.PENDING, Job.Status.STARTED, Job.Status.QUEUED))`. No strong opinion, whatever suits you more. The latter `__in` is more common in the codebase.